doc/develop/west/zephyr-cmds.rst

```diff
@@ -174,3 +174,20 @@ As does deleting them::
 Additionally the tool allows you to specify the modules you want to list,
 fetch or clean blobs for by typing the module names as a command-line
 parameter.
+
+.. _west-twister:
+
+Twister wrapper: ``west twister``
+*****************************************
+This command is a wrapper for :ref:`twister <twister_script>`.  It leverages
+current :envvar:`ZEPHYR_BASE` environment variable and west manifest to launch
+the twister version associated with the west workspace. `west twister` resolves
```

Comment:
You need two backticks for inline code in .rst. The single backtick syntax is markdown.

Reply:
done